__tests__/helpers/EVM.ts

```diff
@@ -15,5 +15,6 @@ export async function snapshot() {
 }
 
 export async function fastForward(seconds: number) {
-  return solo.testing.evm.increaseTime(seconds);
+  await solo.testing.evm.increaseTime(seconds);
```

Comment:
increaseTime in the new ganache-cli will no longer automatically mine a block, so I added it here. Nothing uses the return value of this function so now it doesn't return anything